cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java

```diff
@@ -200,6 +201,9 @@ private void publishNotification(long absoluteSize) {
       try {
         notificationService.publish(streamFeed, new StreamSizeNotification(System.currentTimeMillis(), absoluteSize))
           .get();
+      } catch (ServiceUnavailableException e) {
```

Comment:
Would it be good to propagate the exception if the next iteration doesn't have it fixed (I mean like after N iterations, the issue is still there)?